webapp/src/components/sidebar/__snapshots__/sidebar.test.tsx.snap

```diff
@@ -51,9 +51,9 @@ exports[`components/sidebarSidebar dont show hidden boards 1`] = `
                   >
                     <div
                       class="version"
-                      title="v7.6.0"
```

Comment:
If these version strings in typescript files started using the string in the constants file it would require a lot less file edits.

Reply:
Can you do that with snapshot files? Because they get built by running the tests.

Reply:
Good point.  I realize now those aren't changing by manually editing them - they are changing by updating the snapshots.  My mistake.